### ISSUE TITLE
'Trailing characters' error in command using bar ('|')

### ISSUE THREAD

**Issue by HawkinsT** [vifm/vifm#517]
In powershell on windows 10 with fzf installed, the following command works fine:

`Get-ChildItem -Recurse | % { $_.FullName.Replace("\","/") } | fzf`

It produces a plain text list of all files and folders in the current directory and sub directories and replaces the back slashes with forward slashes, with fzf allowing you to browse this list and select one item which is then output in plain text.

In trying to goto this selected output file in vifm I've created the following command:

`command! FindInDir :execute "goto '".term('Get-ChildItem -Recurse | '.expand('%%').' { $_.FullName.Replace("\","/") } | fzf')."'"`

I've also tried replacing the backslash with "\\\\". Unfortunately, the bar between recurse and expand causes vifm to throw the 'Trailing characters' error which I'm not sure how to avoid as this is required by the powershell command.

While I'm not ruling out this being an error on my part, I notice that in the change log there's the old entry:

0.7.4b to 0.7.5-beta (2013-04-22)
Fixed a bug with trailing characters on loading multiline commands containing bars ('|') from a vifmrc file.

and wonder if this is in fact a related bug.

**Comment by xaizek** [comment]
> the bar between recurse and expand causes vifm to throw the 'Trailing characters' error

If that bar was treated as a :command-separator, you'd get "Invalid expression" error due to first command. "Trailing characters" error likely means that `:goto` command got more than one argument. See what you'll get with this version (I don't get what `expand('%%')` is supposed to mean, by the way):

```vim
command! FindInDir :execute 'goto' fnameescape(term('Get-ChildItem -Recurse | '.expand('%%').' { $_.FullName.Replace("\","/") } | fzf'))
```

**Comment by HawkinsT** [comment]
Thanks. `expand('%%')` was simply me trying to be super verbose in typing `%` as I wasn't sure what the error was. I've now tracked down the issue though thanks to your example (which gives a more helpful error message):

Despite running vifm through powershell, all commands from vifm are being sent to cmd (so powershell commands don't work). I might be able to sidestep this with a script invoking powershell from cmd, so I'll explore this option now. I did quickly try:
`PowerShell -Command "Get-ChildItem -Recurse | % { $_.FullName }" | fzf`
Which works in cmd outside of vifm, but not inside still, but at least I get a blank list instead of an error now so it suggests I might be able to create some sort of powershell script that outputs to cmd for vimf to read.
